virttest/qemu_vm.py

```diff
@@ -4857,7 +4857,7 @@ def restore_from_file(self, path):
         self.create(name=self.name, params=self.params, root_dir=self.root_dir,
                     timeout=self.MIGRATE_TIMEOUT, migration_mode="exec",
                     migration_exec_cmd="cat " + path, mac_source=self)
-        self.resume()
+        self.verify_status('paused')  # Throws exception if not
```

Comment:
Please note that the following test has to be updated accordingly.

https://github.com/autotest/tp-qemu/blob/19113f9526b359c9d6d3c88d436dad6d2b3adde6/qemu/tests/save_restore_vm.py#L43-L44

Reply:
Hmmm, I don't have a clone of the repo from what I recall and perhaps and since a potential update there must be synced with potential update here perhaps is better to ping a maintainer and thus bring to more awareness about the change?

As I have a few more pull requests I might push touching upon the same functionality after sufficient testing and analysis perhaps I can open an issue there to track required updates for the test that I can then propose all at the same time? In short, I am looking for ways to decouple this pull request from a pull request there as much as possible.

Reply:
Ok, opened a todo there about this update: https://github.com/autotest/tp-qemu/issues/3523#issue-1432944738